cypress/integration/admin/fileManager/uploadImage.spec.js

```diff
@@ -1,3 +1,50 @@
+const openFileDetails = index => {
+    cy.waitUntil(
+        () =>
+            cy
+                .findByTestId("fm-list-wrapper")
+                .within(() => {
+                    cy.findAllByTestId("fm-list-wrapper-file")
+                        .eq(index)
+                        .within(() => {
+                            cy.findByTestId("fm-file-wrapper-file-info-icon").click({
+                                force: true
+                            });
+                        });
+                })
+                .then(() =>
+                    cy.findByTestId("fm.file-details.drawer").then($el => {
+                        const [aside] = $el;
+                        return aside.classList.contains("mdc-drawer--open");
+                    })
+                ),
+        {
+            description: `Wait until "File Details" model is visible`
+        }
+    );
+};
+
+const deleteFile = () => {
```

Comment:
Since this is used for cleanup purposes, let's programmatically delete files, and not via UI. Going to be faster. Plus we'll be able to delete `openFileDetails` (I just realized it's only used within `deleteFile`.).

We already have the `fmDeleteFile` command in `cypress/support/fileManager/fmDeleteFile.js`.